sda-commons-dependencies/build.gradle

```diff
@@ -125,6 +123,11 @@ dependencies {
     api "io.prometheus:simpleclient_dropwizard:$prometheusVersion"
     api "io.prometheus:simpleclient_servlet:$prometheusVersion"
 
+    // sda-commons-server-s3
+    api "commons-logging:commons-logging:1.2", {
```

Comment:
Does it include security updates or needed features? I am not sure if we want to upgrade a dependency just for the sake of upgrading it.

Reply:
I selected the version that is in master at the moment. I'm not sure if it has a reason. Maybe some compatibility issues with SLF4J. But we can try the version that comes with the aws lib. The only part from the changelog that may be useful is „Improve performance of LogFactory#getFactory()“. But that depends on how and how often the aws lib uses the LogFactory.